### PR TITLE
Badge server and validation

### PR DESCRIPTION
**Pull request by FrenchBen**
Added a server to show the badge badge when a repo has more than 3 badges.

Signed-off-by: French Ben frenchben@docker.com

### CODE REVIEW

File: validate.go [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+var badgeMinimum int
+
+const inadequateBadge = "badges/inadequate.svg"
+const adequateBadge = "badges/adequate.svg"
+
+var badgeRex = regexp.MustCompile("(?i)(\\[!\\[[a-zA-Z0-9_ ]*\\]\\([0-9a-z.\\/:?=-]*\\)\\]\\([0-9a-z.\\/:-]*\\))")
+
+func checkBadges(username string, reponame string) (string, error) {
+	resp, err := http.Get(fmt.Sprintf("https://raw.githubusercontent.com/%s/%s/master/README.md", username, reponame))
```
Comment (stevvooe) [inline validate.go]: Wish: take a branch.

Comment (FrenchBen) [inline validate.go]: Like travis-CI does? so you can pass `?branch=some-branch`

Comment (stevvooe) [inline validate.go]: Yep, where default is master. I think you can just shove it into this URL at that point.

**Comment by thaJeztah** [comment]
👍 

screenshot-at-2012-03-22-125454

**Pull request merged**